dlf/plugins/pageview/tx_dlf_utils.js

```diff
@@ -561,38 +561,48 @@ dlfUtils.isCorsEnabled = function(imageObjs) {
     // fetch data from server
     // with access control allowed
     let response = true;
-
+    const type = 'HEAD';
     imageObjs.forEach(function(imageObj) {
         let url = imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.ZOOMIFY
-            ? imageObj.url.replace('ImageProperties.xml', 'TileGroup0/0-0-0.jpg')
-            :
-            imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.IIIF
-                ? dlfViewerSource.IIIF.getMetdadataURL(imageObj.url)
+            ? dlfUtils.buildXhrURI(imageObj.url.replace('ImageProperties.xml', 'TileGroup0/0-0-0.jpg'), true)
+            : imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.IIIF
+                ? dlfUtils.buildXhrURI(dlfViewerSource.IIIF.getMetdadataURL(imageObj.url), false)
```

Comment:
I think I get what you are saying, and I now see that there are several problems with the intent of this function (that I guess is to preflight CORS requests for WebGL?).  I agree that the `HEAD` approach does not work to fix it either.
1. The resource url _MUST_ be called by XHR directly and not through the eID.  This is the only way for the browser to get the CORS response headers directly from the server.  
2.  `HEAD ` or `GET` are not a substitute for `OPTIONS` and cannot return values to the client for the CORS response headers (because they are explicitly filtered by XHR).  
3.  A **preflight enabled** CORS server _MUST_ also include a header like this:
`Access-Control-Allow-Methods →HEAD, GET, OPTIONS`.  Without the explicit declaration, a preflight `OPTIONS` is not possible!   

I suspect that the preferred way to check whether or not a server supports CORS in a preflight is to send an `OPTIONS` request that includes this header: `Access-Control-Request-Method:OPTIONS`.  A conforming server would respond with a 200 and a non-conforming one will fail.  I have updated [the patch](https://github.com/kitodo/kitodo-presentation/pull/194/commits/dd67a5ea6386b6e407e9ff9976a11c7f4174c5e2) to reflect this.  My feeling is that the server needs to be preflighted, then the right CORS headers should be implemented by the server in accordance with the specifications, and this definitely includes allowing the access-control `OPTIONS` method.